clients/src/main/java/org/apache/kafka/common/config/AbstractConfig.java

```diff
@@ -360,4 +360,27 @@ public V get(Object key) {
             return super.get(key);
         }
     }
+
+    /**
+     * Postprocess the configuration so that exponential backoff is disabled if the reconnect backoff
+     * is configured, but the maximum reconnect backoff is not configured.
+     *
+     * @param parsedValues          The parsedValues as provided to postProcessParsedConfig.
+     * @param reconnectBackoff      The name of the configuraiton key for reconnect backoff.
+     * @param reconnectBackoffMax   The name of the configuraiton key for maximum reconnect backoff.
+     * @return                      The new values which have been set as described in postProcessParsedConfig.
+     */
+    protected Map<String, Object> postProcessReconnectBackoffConfigs(Map<String, Object> parsedValues,
```

Comment:
Well... it does allow you to parameterize the key names, so it is generic in that sense.

Unfortunately, we need access to `AbstractConfig#originals`, which is currently private.  And I think one or two other private things in `AbstractConfig`.  So we can't put this in a utility at the moment.... but at least it is `protected`, so it is not exposed to end-users and we could change it later.